Tests/SonarQube.TeamBuild.PostProcessor.Tests/CoverageReportProcessorTests.cs

```diff
@@ -0,0 +1,160 @@
+﻿/*
+ * SonarQube Scanner for MSBuild
+ * Copyright (C) 2016-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using SonarQube.Common;
+using SonarQube.TeamBuild.Integration;
+using SonarQube.TeamBuild.Integration.Interfaces;
+using TestUtilities;
+
+namespace SonarQube.TeamBuild.PostProcessor.Tests
+{
+    [TestClass]
+    public class CoverageReportProcessorTests
```

Comment:
@valhristov refactored some of this code in _master_ but some of the refactorings were dropped when merging the _mono_ changes (too difficult to fix everything up). Some of the existing tests were dropped too.
The tests for the _CoverageReportProcessor_ had been deleted. I rescued then and fixed up the ones that still made sense to bump up the code coverage numbers.

We need to do some more clean-up of these classes, but not in this sprint.